### ISSUE TITLE
SVG icon

### ISSUE THREAD

**Issue by ghisvail** [raspberrypi/rpi-imager#66]
Right now, the [icon](https://github.com/raspberrypi/rpi-imager/blob/qml/icons/rpi-imager.png) is a fixed-size PNG file, which is acceptable by today's Linux packaging standards.

It would be nice to provide a SVG version of the icon for desktop environments which support dynamic scaling.

**Comment by lurch** [comment]
@spl237 do you have a suitable SVG?

**Comment by spl237** [comment]
No, I don't use SVG icons anywhere, as they're a pain to create. Might be worth asking the comms team or design if they have one.

**Comment by ghollingworth** [comment]
@jackbenwillis

**Comment by spl237** [comment]
That said, there probably isn't a huge benefit of going to all the hassle of creating an SVG as opposed to just having a few bitmaps at appropriate sizes; unless Jack is starting from vectors anyway...

**Comment by jackbenwillis** [comment]
I generally prefer to work in vector. And only have to make the design once. Although for the new icons Simon and I worked on I was working mostly in bitmap since you get some weird approximations at smaller sizes, which you can remedy by stripping out detail at pixel level.

Which icon do we need a SVG for?

**Comment by headblockhead** [comment]
Sorry to revive an old issue, but I designed my own 'unofficial' SVG version using the raspberry pi logo from raspberrypi.com if anyone wants to use it - see #949 for the lack of my own pull request. 

If anyone, including Raspberry Pi's design team, want to use this file, I give full permission.

(the low-resolution PNG icon really bugs me next to all the other vector icons.)

[new file](https://raw.githubusercontent.com/headblockhead/rpi-imager/refs/heads/qml/rpi-imager.svg) 
<img width=256 src=https://github.com/user-attachments/assets/bd5c2e20-de41-43de-883d-34b1594b250a/>
[old file](https://raw.githubusercontent.com/raspberrypi/rpi-imager/refs/heads/qml/debian/rpi-imager.png)
<img width=256 src=https://github.com/user-attachments/assets/2240638d-afcd-4c96-a7d2-52477256ec93/>
comparison:
<img width=256 src=https://github.com/user-attachments/assets/cdba3e77-3bb7-4bfc-9e69-eb0dacde326f/>

**Comment by ghisvail** [comment]
That's very nice, looks amazing

**Comment by PortalWalkerCodes** [comment]
Looks amazing!

**Comment by nafanz** [comment]
Yeah, it looks good.

**Comment by tdewey-rpi** [comment]
SVG icons are present from 2.0.2.
